Source/WebCore/platform/graphics/ColorInterpolationMethod.cpp

```diff
@@ -26,10 +26,78 @@
 #include "config.h"
 #include "ColorInterpolationMethod.h"
 
+#include <wtf/text/StringBuilder.h>
 #include <wtf/text/TextStream.h>
 
 namespace WebCore {
 
+void serializationForCSS(StringBuilder& builder, ColorInterpolationColorSpace interpolationColorSpace)
```

Comment:
Should have a function that returns an `ASCIILiteral` separate from the one that does the appending.